static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx

```diff
@@ -30,16 +30,16 @@ export function SpanEvidenceKeyValueList({
     },
     {
       key: '2',
-      subject: t('Repeating Span'),
-      value: getSpanEvidenceValue(repeatingSpan),
+      subject: t('Offending Span'),
```

Comment:
This works, although I wonder if we could have a separate function to get the subject fields. I think we'd want to keep them as `Repeating Span` for N + 1 queries, and future issues will likely have different specialized names for their offending span.

Reply:
@0Calories gonna need this to do that cause we don't have issueType in the issue list page:

https://github.com/getsentry/sentry/pull/41901